x-pack/plugins/security_solution/server/search_strategy/event_filters_fields/index.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { from } from 'rxjs';
+import type {
+  DataViewsServerPluginStart,
+  ISearchStrategy,
+  SearchStrategyDependencies,
+} from '@kbn/data-plugin/server';
+
+import { requestIndexFieldSearch } from '@kbn/timelines-plugin/server/search_strategy/index_fields';
```

Comment:
Might be a silly question, but is this even allowed? iirc, importing like this should be done from the files specified in the `extraPublicDirs` of the plugin you're importing from and the `server/` path isn't surfaced. I'm not sure, but maybe someone from @elastic/kibana-operations / @spalger may know? Just don't want us to run into any dependency lifecycle issues in the future

Reply:
On the server side of things there are far less reasons to be concerned. It's when bundling stuff for the front-end that we run into most problems so that's where the majority of rules are.

Reply:
Great, thanks!